final_task/rss_reader/classes/news_class.py

```diff
@@ -16,18 +18,22 @@ class News:
     links: str
     feed_title: str
     source: str
+    command_line_args: "argparse instance"
```

Comment:
It is better to try to think of some other method, instead of storing arguments in each instance of News class.

Reply:
Fixed